docs/artifacts/who-decides-done.html

```
@@ -0,0 +1,273 @@
+
+<h2>What the tool asks, and what it doesn't</h2>
+</div>
+
+<figure>
+<pre class="mermaid">
+%%{init: {'theme':'base','themeVariables':{'primaryColor':'#EDF0F5','primaryTextColor':'#14181F','primaryBorderColor':'#5F6875','lineColor':'#4a4a4a','fontSize':'14px'}}}%%
+flowchart LR
+  F["chain file<br/>21 entries in order"] -->|"gives the ORDER"| T["next-ticket"]
+  T -->|"asks: are the BLOCKERS closed?"| G["GitHub tracker"]
+  G -->|"OPEN / CLOSED per blocker"| T
+  T -.->|"NEVER asks:<br/>is the ENTRY ITSELF closed?"| G
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)